package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "metal-metal-key",
+  "version": "0.0.0",
+  "description": "Adds a custom event that helps listen to key events in a cleaner way",
+  "license": "BSD",
+  "repository": "fernandosouza/metal-metal-key",
+  "engines": {
+    "node": ">=0.12.0",
+    "npm": ">=3.0.0"
+  },
+  "main": "lib/all/metalKey.js",
+  "jsnext:main": "src/MetalKey.js",
```

Comment:
This should point to `src/all/metalKey.js`, otherwise it won't be possible to import `KEYMAP` from `metal-key` when running bundlers like Rollup or gulp-metal's globals task.